main.go

```diff
@@ -330,116 +330,116 @@ func main() {
 		var err error
 		keystorePath, err = pathutil.AbsPath(pth)
 		if err != nil {
-			log.Error("Failed to expand path (%s), error: %s", pth, err)
+			log.Errorf("Failed to expand path (%s), error: %s", pth, err)
 			os.Exit(1)
 		}
 	} else {
-		log.Info("Download keystore")
+		log.Infof("Download keystore")
 		keystorePath = path.Join(tmpDir, "keystore.jks")
 		if err := download(configs.KeystoreURL, keystorePath); err != nil {
-			log.Error("Failed to download keystore, error: %s", err)
+			log.Errorf("Failed to download keystore, error: %s", err)
 			os.Exit(1)
 		}
 	}
-	log.Detail("using keystore at: %s", keystorePath)
+	log.Printf("using keystore at: %s", keystorePath)
 	fmt.Println("")
 
 	keystore, err := keystore.NewHelper(keystorePath, configs.KeystorePassword, configs.KeystoreAlias)
 	if err != nil {
-		log.Error("Failed to create keystore helper, error: %s", err)
+		log.Errorf("Failed to create keystore helper, error: %s", err)
 		os.Exit(1)
 	}
 
 	//
 	// Find Android tools
 	androidHome := os.Getenv("ANDROID_HOME")
-	log.Detail("android_home: %s", androidHome)
+	log.Printf("android_home: %s", androidHome)
 
 	latestBuildToolsDir, err := latestBuildToolsDir(androidHome)
```

Comment:
finding the `latestBuildToolsDir` is common issue, please solve this in go-android package